test/univariate.jl

```diff
@@ -13,6 +13,9 @@
         elseif solver == :SumOfSquares
             # solver returns negative -1.06259e-06
             @test abs(inf(r)) ≤ 1e-5 && sup(r) ≤ 1e-5
+        elseif solver == :BranchandBound
```

Comment:
remove this case? if the solver returns a relative precision of `[1.21721e-15, 0]` there is no need to special case it since the bounds nonnegative (and way smaller than 1e-5)

Reply:
Yes Indeed.